### DISCUSSION TITLE
Question about Bfield traces and batch mode

### DISCUSSION THREAD

**Discussion by chrismarsden7** [plasmapotential/HEAT#35] [Q&A]
Hello! - thought I'd try out the discussions page for asking questions that aren't strictly issues.

I was wondering if it is possible to perform Bfield traces in batch mode. Whenever I perform Bfield traces using HEAT I have presently been using the GUI. I was wondering if it is possible to perform Bfield traces (and not any sort of optical or gyro orbit heat loading calculation) using the TUI instead?

Cheers,
Chris

**Comment by tloobyCFS** [comment]
Hey Chris.  Unfortunately, this is currently not supported.  As you know, you can run multiple traces from a single csv file in GUI mode, but it has not been added in TUI mode.  I think it would be pretty easy to add tho.

This would be a great feature to add if you want to create a fork, mock it up, and then submit a pull request.  If you want a hand figuring out how to do it I would also be happy to help.

-tom
